ingestion-lambda/src/handler.test.ts

```diff
@@ -267,21 +274,78 @@ describe('handler.main', () => {
 		);
 	});
 
-	it('should return early when it receives an SES event', async () => {
+	it('should be able to process an SES event', async () => {
+		mockGetFromS3.mockResolvedValue({
+			status: 'success',
+			body: sampleMimeEmailData(),
+		});
+
+		const mockSql = jest.fn();
+		mockSql.mockResolvedValue([{ id: 1 }]);
+
+		mockInitialiseDbConnection.mockResolvedValue({
+			sql: mockSql as unknown as postgres.Sql,
+			closeDbConnection: jest.fn(),
+		});
+
+		const sesEvent = {
+			Records: [
+				{
```

Comment:
Small point, but might be nice to move the creation of this object and call it something to show it's successful, and then you can make a another one that just varies the spam verdict